gui_stat_reports.py

```diff
@@ -177,10 +173,7 @@ def make_gui_plots( guis, bad_thresh, tstart=0, tstop=DateTime().secs, outdir="p
 
     # Fraction not tracking plus bad status vs Mag
     h=plt.figure(figsize=figsize)
-    trak_frac = ((range_guis['not_tracking_samples']
-		  + range_guis['obc_bad_status_samples'])*1.0
-		 / range_guis['n_samples'])
-    plt.semilogy(range_guis['mag_exp'], trak_frac, 'k.')
+    plt.semilogy(range_guis['mag_aca'], (1.0 - range_guis['f_track']) + range_guis['f_obc_bad'], 'k.')
```

Comment:
I have forgotten if we only have bad status flags on a tracked star.  If that is true, this sum of the not-tracking fraction and the obc_bad fraction would be OK for the "or" of the two sets.  If we can be not tracking while flags are set, there would be some overlap in the sets.